src/ScrumMaster/Channel/Email/MessageGenerator.php

```diff
@@ -33,20 +33,19 @@ public function forJiraTicket(JiraTicket $ticket, string $companyName): string
         $dayWord = ($daysDiff > 1) ? 'days' : 'day';
 
         return $this->headerText($assignee) . <<<TXT
-Send via EMAIL :)
-*Ticket*: {$ticket->title()}[<{$url}|{$ticket->key()}>]
-*Current status*: {$status->name()} since {$daysDiff} {$dayWord}
-*Story Points*: {$ticket->storyPoints()}
-
+Send via EMAIL :) <br>
+*Ticket*: {$ticket->title()}[{$url}|{$ticket->key()}] <br>
+*Current status*: {$status->name()} since {$daysDiff} {$dayWord} <br>
+*Story Points*: {$ticket->storyPoints()}<br>
 TXT;
     }
 
     private function headerText(Assignee $assignee): string
     {
         if ($assignee->key()) {
-            return "Hey, {$assignee->displayName()} ({$assignee->name()}), please have a look" . PHP_EOL;
+            return "Hey, {$assignee->displayName()} ({$assignee->name()}), please have a look <br>";
         }
 
-        return 'Hey Team, please have a look' . PHP_EOL;
+        return 'Hey Team, please have a look <br>';
```

Comment:
Why do you use here HTML code instead of `PHP_EOL`?

Reply:
The idea was using a html template for the email template. Is not working for some reason. Anyway, I will extract this particular logic to a follow up issue to improve the rendering of the emails